### ISSUE TITLE
regarding builtin_function_or_method

### ISSUE THREAD

**Issue by vainaijr** [pytorch/pytorch#25381]
## 🚀 Feature

some way to see the code for builtin_function_or_method
## Motivation

consider this scenario
1) I am using google colab, I want to see the implementation of nn.Dropout
2) I ctrl+click on nn.Dropout, it takes me to dropout.py, where I find Dropout class, which returns
`
F.dropout(input, self.p, self.training, self.inplace)
`
3) so, I ctrl+click on F.dropout, it takes me to functional.py, where I see this
`
return (_VF.dropout_(input, p, training)
            if inplace
            else _VF.dropout(input, p, training))
`
4) now, I am stuck, because this is a builtin, and only way for me to look at its code, is to search for this in github.
5) same thing happens when I want to see the implementation of nn.CrossMapLRN2d, it redirects to torch.group_norm, and it is builtin.
6) similarly for view, permute, this way to understand what these builtins are doing becomes difficult

because of this I raised an issue in google colab, to provide a way to look at builtin_function_or_method, they told 

`
No, there's no way to get directly to the source in this case.

In particular, for a .so, it could be a cython file, or it could be another kind of C extension (built via vanilla C/C++ code, or SWIG'd in, or something else completely). All we have is the installed package -- there's no way for us to trace that back to source, and in cases like this (torch is distributed as a .whl file) the source isn't included.
`

**Comment by VitalyFedyunin** [comment]
Unlikely we will ever find resources to implement this feature. I'm leaving it open in case of open source community contributions.
